chart/values.yaml

```diff
@@ -446,15 +446,13 @@ opentelemetryOperator:
             exporters: [logging, otlp]
             processors: [batch]
 
-  jaegerPromscaleQuery:
-    image: timescale/jaeger-query-proxy:0.0.1
-    endpoint: "{{ .Release.Name }}-jaeger-promscale.{{ .Release.Namespace }}.svc.cluster.local:16686"
+  jaeger:
+    image: jaegertracing/jaeger-query:1.30
+    endpoint: "{{ .Release.Name }}-promscale-connector.{{ .Release.Namespace }}.svc.cluster.local:9202"
```

Comment:
This line reads a bit weird now, because it's not the endpoint of the jaeger query image above. It's only really used by Grafana. So maybe move it up into the Grafana section? An example of the awkwardness: what if you wanted to disable the jaeger query ui but keep the Grafana datasource setup.